antiplag/tasks.py

```diff
@@ -41,6 +46,18 @@ def process_documents(submission_id):
     submission.status = Submission.SubmissionStatus.PROCESSED
     submission.save()
 
+    # send email when done
+    if submission.email is not None:
+        try:
+            validate_email(submission.email)
+        except ValidationError as e:
+            print(_("bad email, details:"), e)
```

Comment:
Please make this message more formal. What about `Unrecognized email address format. Error:`?

Reply:
Also, what kind of error is in here? Aren't we exposing some vulnerable information about or system? e.g. stacktrace or so.